src/libstd/panicking.rs

```diff
@@ -368,7 +368,8 @@ fn default_hook(info: &PanicInfo) {
     let name = thread.as_ref().and_then(|t| t.name()).unwrap_or("<unnamed>");
 
     let write = |err: &mut ::io::Write| {
-        let _ = writeln!(err, "thread '{}' panicked at '{}', {}:{}:{}",
+        // Add a cariage return to reset cursor to the beginning of the line with rawmode tty's
+        let _ = write!(err, "thread '{}' panicked at '{}', {}:{}:{}\r\n",
```

Comment:
Would it be better to do `"thread '{}' panicked at '{}', {}:{}:{}\r\n"` instead?

Reply:
Isn't that what it is now?

Reply:
oops, my bad, wrong copy & paste. I meant `"\r\nthread '{}' panicked at '{}', {}:{}:{}"`